lib/index.js

```diff
@@ -153,9 +153,9 @@ var Falafel = function () {
 
 			}
 
-
+			const disableDevServer = options.disableDevServer || process.env.FALAFEL_DISABLE_DEV_SERVER;
```

Comment:
`process.env.FALAFEL_DISABLE_DEV_SERVER` - you'll need to convert this into a boolean